CHANGE_LOG

```diff
@@ -19,6 +19,12 @@ Pull-Requests:
 * PR `#8411 <https://github.com/numba/numba/pull/8411>`_: update version support table for 0.56.1 (`esc <https://github.com/esc>`_)
 * PR `#8412 <https://github.com/numba/numba/pull/8412>`_: Create changelog for 0.56.1 (`Andre Masella <https://github.com/apmasell>`_)
 * PR `#8413 <https://github.com/numba/numba/pull/8413>`_: Fix Azure CI for NumPy 1.23 and use conda-forge scipy (`Siu Kwan Lam <https://github.com/sklam>`_)
+* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
+
+Version 0.56.1 (1 September, 2022)
```

Comment:
Should this also say "NO RELEASE" for the release date?

Reply:
probably, yes

Reply:
added in bcfef8d